datayoga/blocks/filter/tests/test_filter.py

```diff
@@ -0,0 +1,25 @@
+from datayoga.blocks.filter.block import Block
+
+
+def test_filter_sql():
+    block = Block({"language": "sql",
+                   "expression": "age>20"})
```

Comment:
can be in one line or the assert should be in the same format (field under field), same for the rest of the tests.